libass/ass_bitmap.c

```diff
@@ -370,13 +370,15 @@ void be_blur_post(uint8_t *buf, intptr_t w, intptr_t h, intptr_t stride)
  */
 int be_padding(int be)
 {
+    if (be <= 0)
+        return 0;
     if (be <= 3)
-        return be;
+        return be + 1;
     if (be <= 7)
-        return 4;
-    if (be <= 123)
         return 5;
-    return FFMAX(128 - be, 0);
+    if (be <= 123)
+        return 6;
+    return FFMAX(128 - be, 0) + 2;
```

Comment:
In other news, just as my own comment above this function already says, at huge \be values, enough padding is needed for all intermediate values, so I think the last line with the `FFMAX` is wrong, and it should just return 5 (before your commit, or 6 after) in the default case. Of course, I wrote this code so it’s my own bug, if that’s right. I’m not sure what I was thinking, _especially_ because the comment clearly notes that intermediate values need large padding.

Reply:
Removed this commit.